slash_commands/index-2mp.js

```diff
@@ -30,6 +28,19 @@ builder = new SlashCommandBuilder()
     .addStringOption(userOption)
     .addStringOption(reloadOption);
 
+async function fetch2mp(searchParams) {
+    let res = await fetch('https://btd6index.win/fetch-2mp?' + searchParams);
+    let resJson = await res.json();
+    if ('error' in resJson) {
+        throw new Error(resJson.error);
+    }
+    return resJson;
+}
+
+function genCompletionLink(completion) {
+    return `[Link](${completion.link ?? `https://media.btd6index.win/${completion.filekey}`})`;
```

Comment:
I feel like it'd be helpful to tell the user where they're going. [Here's what I did](https://github.com/hemisemidemipresent/cyberquincy/blob/9e0471fd80afc2841e2c1bab74566a7b39ef15b8/slash_commands/index-fttc-new.js#L258-L266). I think you just want to combine these two approaches and replace `Link` here with some case options as presented in my PR.